src/myparcelcom.css

```diff
@@ -8,6 +8,12 @@ img[alt=logo] {
   margin: 20px 0 !important;
 }
 
-redoc {
-  display: block;
+/* avoid long regex to overlap the right area */
+.api-content > div > div:first-child > div:first-child {
+  overflow-x: scroll;
```

Comment:
I believe that this breaks the design somehow (check the x-scrolls on the image below):

![image](https://user-images.githubusercontent.com/46448148/84143873-99f3f480-aa57-11ea-9d61-11a95af961fa.png)

Reply:
Will change it to `auto` so Chrome does not show bars by default.